metron-platform/metron-solr/README.md

```diff
@@ -95,3 +95,19 @@ As of now, we have mapped out the Schemas in `src/main/config/schema`.
 Ambari will eventually install these, but at the moment it's manual and
 you should refer to the Solr documentation [https://lucene.apache.org/solr/guide/6_6](here) in general
 and [here](https://lucene.apache.org/solr/guide/6_6/documents-fields-and-schema-design.html) if you'd like to know more about schemas in Solr.
+
+In Metron's Solr DAO implementation, document updates involve reading a document, applying the update and reindexing the whole document.  
+Indexing LatLon and PointType field types stores data in internal fields that should not be returned in search results.  For these fields a dynamic field type matching the suffix needs to be added to store the data points.
```

Comment:
We went through a lot of discussion on this PR to figure out the exact fields and versions of Solr that the LatLon and Point types are problematic for. Can you be a bit more explicit about that here? It's not just "for example" it's literally "these types are a problem bc \<link to problem\> in Solr version X.X, but have been corrected in Solr version X.Y"

Reply:
This problem exists for these types in 5.5 and 6.6.  I added a comment that recommends using a different spatial field type if on version 6+.